src/libcollections/borrow.rs

```diff
@@ -66,7 +66,9 @@ pub trait ToOwned {
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl<T> ToOwned for T where T: Clone {
+impl<T> ToOwned for T
+    where T: Clone
+{
```

Comment:
(rustfmt bug again)

I think this is intentional though, since the where clause was indented